### DISCUSSION TITLE
下書きの編集再開ボタンが分かりづらい

### DISCUSSION THREAD

**Discussion by keisuke-nakata** [increments/qiita-discussions#526] [要望・アイデア・不具合]
# 問題の説明

<img width="1678" alt="image" src="https://github.com/increments/qiita-discussions/assets/6150143/1bc359d5-56b6-45ca-b2a7-bc90fc34f34f">

下書き一覧ページから、下書きの編集を再開しようとしたところ、編集再開ボタンがどこにあるかわからずしばらくさまよってしまいました。
↑の画像の例ではまだ気づきそうですが、大きな画面で表示している場合、(記事の横幅がレスポンシブではなく固定であるため) 両端に広大な余白が表示されますが、編集再開ボタンは右下に表示され続けるためさらに気づきにくいです。

# 改修案

- 今のボタンを目立たせる
  - ボタンそのものを大きくする
  - アイコンだけではなく、「編集再開」「下書き削除」等の文字も同時に表示する
    - アイコンのみのボタンは、基本的には推奨されないのが最近の風潮とも思います
    - 右上の「投稿する」は文字が併記されています
  - 編集再開ボタンの色を灰色から緑色にする
    - なぜ今灰色なのかイマイチわかりません。灰色は「削除」や「公開停止」のようなネガティブな印象が一般にあると感じますので、「編集再開」にはあまりふさわしくないのでは。
    - 右上の「投稿する」が緑色なので、ほぼ同じ画面に遷移する「編集再開」も緑色が自然ではないでしょうか？
- ボタンのフローティングをやめる
  - 下書き記事表示領域の上のほうに表示、でも良いのでは。
  - 下書き一覧を見に来た人が何をしたいかを考えると、大抵は下書きを読みたいわけではなく、編集再開か下書き削除だと思いますので、一番目立つ場所にあっても良いくらいだと感じます

**Comment by wataru86** [comment]
@keisuke-nakata

ご意見ありがとうございます🙏 ご不便をおかけして、申し訳ありません。
下書き一覧ページの改善は運営としてもぜひやりたいと思っております！✨
直近の開発優先度などから着手はできておりませんが、検討します！

**Comment by wataru86** [comment]
@keisuke-nakata

お待たせいたしました :bow: 
下書き一覧ページのデザインを変更し、編集ボタンと削除ボタンを左サイドバーに移動しました！
https://qiita.com/release-notes/2023-10-05

**Reply by shiracamus** [reply]
リンク先が Not Found です。

**Reply by tanaka-asami415** [reply]
@shiracamus 
教えていただきありがとうございます！
間違えて該当のリリースノートを削除しておりました🙇 現在は閲覧いただける状態です。
